### ISSUE TITLE
model comparison notebook

### ISSUE THREAD

**Issue by jseabold** [statsmodels/statsmodels#1595]
Do we have a notebook that does model comparison? If not, might be nice to cook one up showing how to use the methods provided by, say, the linear regression models.

**Comment by maobrien11235** [comment]
I'm pretty green to statsmodels, but I would enjoy working on this as my first PR. Would the model comparison be for OLS, WLS, and GLS that are mentioned in the documentation (http://statsmodels.sourceforge.net/devel/regression.html), or are you meaning more of the logistic versus linear regression when you say model comparison?

At any rate, look forward to hearing from ya.

**Comment by kshedden** [comment]
Matt,

The issue is a bit old and there are now some notebooks covering this topic
at the following two locations:

http://kshedden.github.io/

https://github.com/statsmodels/statsmodels/wiki/Examples

However these could definitely use some cleanup, and there are many, many
topics and techniques that are not covered in notebooks. So lots of
opportunities to contribute there.

For example, here is one that gets into AIC a bit:

http://nbviewer.jupyter.org/urls/umich.box.com/shared/static/vlrazg0wglamfqeqhp4y.ipynb

Kerby

**Comment by maobrien11235** [comment]
Good point. I see what cleanup and edits that may be needed in the links you provided. If there isn't anything major, can I swing back around to this page and get this issue closed out?

Cheers.

**Comment by kshedden** [comment]
It doesn't hurt to leave the issue open, these notebooks that I pointed you
to aren't even part of the statsmodels distribution yet.  We could use some
help migrating notebooks into the official documentation, and writing new
documentation notebooks.

Kerby

**Comment by maobrien11235** [comment]
Ah... then I have a lot to learn. I'll get to work. Thanks for clarifying things for me.

**Comment by TomAugspurger** [comment]
@maobrien11235 that'd be great if you want to get those into the official docs. The notebooks go [here](https://github.com/statsmodels/statsmodels/tree/master/examples/notebooks) and are built with [this](https://github.com/statsmodels/statsmodels/blob/master/tools/nbgenerate.py) script to execute the notebooks and convert to HTML. I've got a bit more documentation [here](https://github.com/statsmodels/statsmodels/tree/master/docs). Just ping me if you have any troubles.

**Comment by feliciov** [comment]
This issue has been cold for a few years now. I noticed that the [notebooks](https://github.com/statsmodels/statsmodels/tree/master/examples/notebooks) have been getting more recent attention, so I don't know how relevant It still is. Is there some specific notebook that should be created/update?

**Comment by LeonxLJX** [comment]
Hi! I'd like to work on this issue. Please let me know if I can be assigned. Thanks!

**Comment by bashtage** [comment]
Please do work on it.
